tests/manager/manager-executeBulkWrite_error-002.phpt

```diff
@@ -32,7 +32,7 @@ var_dump(iterator_to_array($cursor));
 ===DONE===
 <?php exit(0); ?>
 --EXPECTF--
-BulkWriteException: BulkWrite error :: E11000 duplicate key error %s: phongo.manager_manager_executeBulkWrite_error_002%sdup key: { : 1 } :: E11000 duplicate key error %s: phongo.manager_manager_executeBulkWrite_error_002%sdup key: { : 2 }
+BulkWriteException: Multiple write errors: "E11000 duplicate key error %s: phongo.manager_manager_executeBulkWrite_error_002%sdup key: { : 1 }", "E11000 duplicate key error %s: phongo.manager_manager_executeBulkWrite_error_002%sdup key: { : 2 }"
```

Comment:
It's a compromise implemented for performance reasons, I imagine. We can certainly ask, but I don't think it's a deal breaker, since the real messages are in the WriteResult object. 500+ bytes is actually quite large when it comes to logging an exception message, and that would easily fit a handful of error messages before truncation.